src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -96,6 +97,18 @@ public boolean contains(ReadOnlyPerson toCheck) {
         return false;
     }
 
+    /**
+     * Sorts persons in address book alphabetically.
+     */
+    private void sortPersonsInAddressBook(){
+        Collections.sort(internalList, new Comparator<Person>(){
+            @Override
+            public int compare(Person o1, Person o2) {
+                return o1.getName().fullName.compareTo(o2.getName().fullName);
```

Comment:
for code readability, you may wish to declare two separate strings for fullName 1 and 2.